src/Files/TemporaryFileFactory.php

```diff
@@ -46,7 +46,7 @@ public function make(string $fileExtension = null): TemporaryFile
      */
     public function makeLocal(string $fileName = null, string $fileExtension = null): LocalTemporaryFile
     {
-        if (!file_exists($this->temporaryPath) && !mkdir($concurrentDirectory = $this->temporaryPath) && !is_dir($concurrentDirectory)) {
+        if (!file_exists($this->temporaryPath) && !mkdir($concurrentDirectory = $this->temporaryPath, config('excel.temporary_files.local_permissions.dir', 0777)) && !is_dir($concurrentDirectory)) {
```

Comment:
I don't think 0777 is a sensible default, I think `0755` would be better

Reply:
This code overrules the default mkdir default, which is 0777 (see [php documentation](https://www.php.net/manual/en/function.mkdir.php)). The default 0777 assures the same behavior as before the PR.

Reply:
Wouldn't it then not be better to just not pass a default here (just `null`) ?

Reply:
The default is 0777, not null (see documentation)

Reply:
I'm still a bit skeptical to put something to 0777. I think even if it's the PHP default we can set the more sensible default of 755.

Reply:
I think it's important to keep the same value as the previous version of the package, which has PHP's default value; if a user upgrades to a newer release the behavior should remains the same (even if it's less safe), or it might break his application.

An option would be to change the default and add it to the release notes; but then one would have to _read_ it...